e2e-tests/monitoring-2-0/conf/secrets.yml

```diff
@@ -11,6 +11,8 @@ data:
   MONGODB_CLUSTER_MONITOR_USER: Y2x1c3Rlck1vbml0b3I=
   MONGODB_USER_ADMIN_PASSWORD: dXNlckFkbWluMTIzNDU2
   MONGODB_USER_ADMIN_USER: dXNlckFkbWlu
+  MONGODB_DATABASE_ADMIN_USER: ZGF0YWJhc2VBZG1pbg==
```

Comment:
I think we should not have it now. But up to you.

Reply:
I have removed it. Let's wait for the tests.